yb-voyager/src/issue/dml.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package issue
+
+var advisoryLocksIssue = Issue{
+	Type:            ADVISORY_LOCKS,
+	TypeName:        "Advisory Locks",
+	TypeDescription: "",
+	Suggestion:      "",
+	GH:              "",
+	DocsLink:        "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#advisory-locks",
```

Comment:
I think we should keep the links in constants still. Also, links need to be changed.

Reply:
WIll change links, but why constants? 
I only put the `type` in constants, in case we want to refer to the issue from some other package, but DocsLink should only be accessed via the issue instance, so why constant?

Reply:
Hmm, sounds right it's not a problem as this place is for just keeping issue structs and these links will not interfere directly with any other code so these are in some way constant data only.

Reply:
Yeah exactly! I'm looking at these issue structs essentially as constants!